docs/settings/triggers/snowboy.md

```diff
@@ -4,23 +4,44 @@ You can create your magic word by connecting to [Snowboy](https://snowboy.kitt.a
 
 Once downloaded, place the file in your personal config folder and configure snowboy in your [your settings](../settings.md) following the table bellow
 
-| parameter   | required | type   | default | choices         | comment                                                                                          |
-|-------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
-| pmdl_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
-| sensitivity | FALSE    | string | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| parameter      | required | type   | default | choices         | comment                                                                                          |
+|----------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
+| keyword_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
```

Comment:
This table should only contains the `keywords` that point to a type 'dict' which is then detailed in another table.
This other table would contains the description of one keyword object with path and sensitivity flags.